frontend/components/account/js/account-modelsetting.component.ts

```diff
@@ -90,6 +90,13 @@ class AccountModelSettingController implements ng.IController {
 
 	}
 
+	public newTopicType(chip) {
```

Comment:
is this used anywhere? 
I'm struggling to find the reference

Reply:
L47 in  frontend/components/account/pug/account-modelsetting.pug  - it's an Angular function responsible for creating the new topic type for the array